scaaml/capture/scope/lecroy/lecroy.py

```diff
@@ -166,22 +187,24 @@ def capture(self, poll_done: bool = False) -> bool:
 
         try:
             # Wait for trigger
-            for _ in range(40):
-                time.sleep(0.1)
+            for _ in range(10):
+                time.sleep(self._trigger_timeout / 10)
                 status = self._scope_communication.query("TRMD?")
                 if status == "STOP":
                     break
             assert self._scope_communication.query("TRMD?") == "STOP"
 
             # Get trace
-            wave = self._scope_communication.get_waveform(channel=self._channel)
+            wave = self._scope_communication.get_waveform(
+                channel=self._trace_channel)
             self._last_trace = wave.get_wave1(
                 first_sample=self._offset,
                 length=self._samples,
             )
 
             return False  # No need to recapture.
         except LeCroyCommunicationError:
+            traceback.print_exc()
```

Comment:
I'm not sure it's wise to print. We use `tqdm` for progress bars which typical requires to use `.write()` on the progress bar to avoid messing with the rendering. And it could make it harder to use the package in python notebooks.

Reply:
Removed, https://github.com/google/scaaml/issues/131